Telerik.JustMock/Core/Behaviors/DelegatedImplementationBehavior.cs

```diff
@@ -37,7 +37,20 @@ public void Process(Invocation invocation)
 			var mockMethod = invocation.Method;
 			var inheritanceChain = mockMethod.GetInheritanceChain();
 
-			var delegatedImplMethod = inheritanceChain.FirstOrDefault(method => types.Any(t => method.DeclaringType.IsAssignableFrom(t)));
+			var delegatedImplMethod =
+				inheritanceChain.FirstOrDefault(
+					method =>
+						types.Any(
+							type =>
+								{
+									var targetType =
+										method.IsExtensionMethod()
```

Comment:
Please refactor the spaces in the ternary operator usage according to our latest discussion on the topic regarding the ternary operator usage for the DebugWindow.

Reply:
@mihail-vladov The diff shown by GitHub the code is somehow mangled, in real it looks much better and according to common practice.